core-kotlin-modules/core-kotlin-advanced-2/README.md

```diff
@@ -8,3 +8,4 @@ This module contains articles about advanced topics in Kotlin.
 - [Get the Type of a Variable in Kotlin](https://www.baeldung.com/kotlin/find-type-variable)
 - [Serialize Kotlin Data Class With Default Values Into JSON](https://www.baeldung.com/kotlin/data-class-json-serialize-default-values)
 - [How to Serialize Arrays in Kotlin With Kotlinx.serialization](https://www.baeldung.com/kotlin/kotlinx-serialization)
+- [Kotlin Generic Inline Classes]()
```

Comment:
It's safe to leave this off, as the core team will add this when they process the article after publication.